packages/slate-editor/src/components/EditorBlock/EditorBlock.tsx

```diff
@@ -72,6 +80,16 @@ export const EditorBlock = forwardRef<HTMLDivElement, Props>(function (
     },
     ref,
 ) {
+    if (typeof children !== 'undefined') {
```

Comment:
Why do we need runtime check, TS will throw an error if you pass some non-undefined children

Reply:
No, it won't.

This is because `children` is declared as `any` in `RenderElementProps` from `slate-react`. 

`any` passes all checks, including `never` and `undefined`.

Reply:
why not just remove `children` from props type declaration?

Reply:
Worked! Thanks! :trophy: 

45293b6ca35046e8aaccf0dde6913d2ea5a114e9